base/cvd/cuttlefish/host/commands/cvdalloc/interface.h

```diff
@@ -20,7 +20,14 @@
 namespace cuttlefish {
 
 constexpr std::string_view kCvdallocInterfacePrefix = "cvd-pi";
+constexpr char kCvdallocMobileIp[] = "192.168.144";
```

Comment:
nit: This is not an IP address, it's a prefix of one. Consider renaming as `kCvdallocMobileIpPrefix`.